.github/workflows/verify-macos.yml

```diff
@@ -14,14 +14,24 @@ jobs:
       # it to verify the Dafny code. Instead we manually pull the submodule we DO need.
       - run: git submodule update --init libraries
 
+      # .NET is already installed, but we need a newer version for dafny-reportgenerator
+      - name: Set up .NET
+        uses: actions/setup-dotnet@v1
+        with:
+          dotnet-version: 6.0.x
+
       - name: Setup Dafny
         uses: dafny-lang/setup-dafny-action@v1
         with:
           dafny-version: "3.5.0"
 
+      - name: Install dafny-reportgenerator
+        run: dotnet tool install --global dafny-reportgenerator
+
       - name: Verify Dafny code
         # Currently, test depends on src, so verifying test will also verify src
-        run: dotnet build -t:VerifyDafny -p:VerifyDafnyJobs=2 -p:TestVerifyOverride="verificationLogger:trx" test
+        run: dotnet build -t:VerifyDafny -p:VerifyDafnyJobs=2 -p:TestVerifyOverride="verificationLogger:csv" test
```

Comment:
Why change it to `csv`? I think that this file is uploaded as an artifact to CodeBuild

Reply:
CSV is what `dafny-reportgenerator` needs as input. It's possible to enable both formats of output, but I think we'd have to invoke `dafny` directly instead of using `dotnet build` for that to work, since I believe it's only possible to specify one `-p:TestVerifyOverride` argument.

On a related note, I've tried verifying the code base with Dafny 3.7.1, on macOS, and it succeeds for me when I directly invoke Dafny but not when I use `dotnet build`, strangely enough. With `dotnet build`, it eventually hangs, using no CPU. I think I've seen some concurrency issues in `dotnet` on macOS in other contexts to, and wonder whether something similar is showing up here.

The bottom line is that it might be worth running `dafny` directly here. That could solve both issues.